src/Models/Customer/Customer.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+namespace Billogram\Api\Models\Customers;
+
+use Billogram\Api\Models\Customer\CustomerContact;
+
+class Customer
+{
+    /**
+     * @var int
+     */
+    private $customerNo;
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var string
+     */
+    private $notes;
+
+    /**
+     * @var string
+     */
+    private $orgNo;
+
+    /**
+     * @var string
+     */
+    private $vatNo;
+
+    /**
+     * @var CustomerContact
+     */
+    private $contact;
+
+    /**
+     * @var CustomerBillingAddress
+     */
+    private $address;
+
+    /**
+     * @var CustomerDeliveryAddress
+     */
+    private $deliveryAddress;
+
+    /**
+     * @var string
+     */
+    private $companyType;
+
+    /**
+     * Customer constructor.
+     *
+     * @param int                     $customerNo
+     * @param string                  $name
+     * @param string                  $notes
+     * @param string                  $orgNo
+     * @param string                  $vatNo
+     * @param CustomerContact         $contact
+     * @param CustomerBillingAddress  $address
+     * @param CustomerDeliveryAddress $deliveryAddress
+     * @param string                  $companyType
+     */
+    public function __construct($customerNo, $name, $notes, $orgNo, $vatNo, CustomerContact $contact, CustomerBillingAddress $address, CustomerDeliveryAddress $deliveryAddress, $companyType)
```

Comment:
Also, since this is used when creating a customer, we may not want to put all things in the constructor.